manage/import-export.js

```diff
@@ -148,7 +148,9 @@ function importFromString(jsonString) {
         !item ||
         !item.name ||
         !item.name.trim() ||
-        (item.sections && !Array.isArray(item.sections))) {
+        !item.sections ||
+        !Array.isArray(item.sections)
```

Comment:
Do we want to check `item.sections.length`?

Reply:
Probably. But then if someone creates a style as follows:

```js
[
    {
        "name": "New Style",
        "sections": [{}]
    }
]
```

It'll still break the editor. Should we add an additional check that `item.sections` contains an object and then check for `item.sections.code`?